docs/changes/wcs/11549.feature.rst

```diff
@@ -0,0 +1,3 @@
+Updated bundled version of WCSLIB to version 7.6. See
+https://www.atnf.csiro.au/people/mcalabre/WCS/CHANGES for a list of
```

Comment:
```suggestion
cextern/wcslib/CHANGES for a list of
```
Last tiny note – might be more convenient to point to the local copy. Don't know if there is a way to correctly link into the source path for the docs; I suppose `~astropy/cextern/wcslib/CHANGES` would do something different.

Reply:
Let me see. I think I did something like this before but I need to find the correct incantation.

Reply:
@dhomeier @pllim  Sorry for the delay - got side-tracked by other things. I got this to work (linking to a local file) but I do not really like how it works: essentially it is a hyperlink to the text file and if you click on it you will have "download" the file before you can look at it. I find it more convenient to link to a web page.

Here are the options related to this (and I do not have hard feelings about how this is done - whatever most people agree and have this in a consistent way, at least in the changelog):
1. Leave as is;
2. Let users "download" WCSLIB changelog;
3. Remove xref completely.

Reply:
I just committed option `#2` - let's just have something different.

Reply:
Hmm, there's no way to put it in  without the `:download:` part? Yes, I suspect most people will prefer a directly readable page, even if it's remote, but it's nice to see that the documentation is available locally; thanks!

Reply:
I doubt a lot of people read the change log diligently, so I am okay with whatever you think is acceptable, as long as it does not break the doc build.

Reply:
> Hmm, there's no way to put it in without the `:download:` part?

I could not figure out how to do it given that WCSLIB `CHANGES` file has no extension. So, `#1` or `#2`.

Reply:
OK. reverted to external web page link

Reply:
Bit like Safari sometimes insists if can only download files it does not recognise as text even though they are clearly readable text... disappointing it depends so much on extensions. :(

Reply:
I think when no extension is provided reST assumes that extension is `.rst`. The only way to get this to work would be to rename `cextern/wcslib/CHANGES` to `cextern/wcslib/CHANGES.txt`. If you do ever find out a way of doing this - feel free to make a PR and ping me too on this.